pkg/asset/manifests/proxy.go

```diff
@@ -95,6 +95,11 @@ func (p *Proxy) Generate(dependencies asset.Parents) error {
 // createNoProxy combines user-provided & platform-specific values to create a comma-separated
 // list of unique NO_PROXY values. Platform values are: serviceCIDR, podCIDR, localhost,
 // 127.0.0.1, api.clusterdomain, api-int.clusterdomain, etcd-idx.clusterdomain
+// and 169.254.169.254 for aws, openstack, gcp and azure instance metadata:
```

Comment:
We also have vSphere and None.. ?

Reply:
> We also have vSphere and None.. ?

Right...but neither have instance metadata via an ip address (at least by default).  I don't think setting that statically would be a problem since I don't see a scenario that you would ever want to proxy a 169 address but I am also ok with checking the platform before setting it as well.

Reply:
Alternative to this PR: https://github.com/openshift/installer/pull/1945